src/main/java/com/launchdarkly/client/Variation.java

```diff
@@ -39,18 +39,17 @@ public boolean equals(Object o) {
     Variation<?> variation = (Variation<?>) o;
 
     if (weight != variation.weight) return false;
-    if (!value.equals(variation.value)) return false;
```

Comment:
I deleted the old equals() and hashCode() methods and told IntelliJ to autogen new ones with null checking.